src/pybind/mgr/dashboard/health.html

```diff
@@ -366,6 +366,7 @@
                             <th>PG status</th>
                             <th>Usage</th>
                             <th>Activity</th>
+                            <th>IOPS</th>
```

Comment:
I prefer to avoid calling the ceph operation counts "IOPS" to avoid confusion with traditional block device IOPS.  Just "ops" perhaps?